src/services/rfqm_service.ts

```diff
@@ -270,9 +271,18 @@ export class RfqmService {
             opts,
         );
 
+        const firmQuotesWithCorrectChainId = firmQuotes.reduce((result, quote) => {
+            if (quote.order.chainId !== CHAIN_ID) {
+                logger.debug(`Received a quote with incorrect chain id: ${quote}`);
```

Comment:
Just want to make sure we want this at the `debug` level

Reply:
this should always be a scary `error` imo. It should never happen, and if it does we (or a market maker) should adjust immediately.

Reply:
done